### DISCUSSION TITLE
http://127.0.0.1:4000/software/2023/10/29/jekyll-and-github-page.html

### DISCUSSION THREAD

**Discussion by giscus** [SuperSharkler/supersharkler.github.io#6] [Announcements]
# http://127.0.0.1:4000/software/2023/10/29/jekyll-and-github-page.html

I have been thinking of building my website for a while, though I was always blocked by my hesitation and uncertainty - how does it work? How will the DB schema look like, what backend API should it support, how should I make sure that only I have the credentials to update the website, etc?

http://127.0.0.1:4000/software/2023/10/29/jekyll-and-github-page.html

**Comment by robosharkler** [comment]
Test agin

**Comment by robosharkler** [comment]
the discussion mapping does not work well in whole URL, as it is possible that I want to change my Github username again in future, which would change the post URL, resulting losing all the comments of each post if using URL as discussion mapping
